kitchen.recipes.yml

```diff
@@ -397,3 +397,22 @@ suites:
     attributes:
       dependencies:
         - recipe:aws-parallelcluster-install::chrony
+  - name: slurm_installed
+    run_list:
+      - recipe[aws-parallelcluster::add_dependencies]
+      - recipe[aws-parallelcluster-slurm::install_slurm]
+    verifier:
+      controls:
+        - slurm_installed
+        - slurm_user_and_group_created
+        - slurm_licence_configured
+        - slurm_library_shared
```

Comment:
This is probably an extra task, but we may want to have a further test that parses the `config.log` produced by the the Slurm configure step to check that Slurm is built with all the functionalities we expect to be there.

If we don't want to address this right now, please create a task for it.

Reply:
Yes. Improving the current recipes is beyond the scope of the task...but, I agree we should add all the required checks. I'll create a task for this.